style.css

```diff
@@ -123,6 +168,45 @@ label{
     background-color: white;
 } 
 
+/* Utility classes */
 .no-border{
     border : none;
 }
+.show{
+    animation: crawlIn 1s ease forwards;
+}
+.hide{
+    animation: crawlOut 1s ease forwards;
```

Comment:
if you have noticed. when the user clicks on **info** icon to show the instuctions and then click on it another time to hide them. the **number of rows ** & **number of columns** inputs doesn't work because you didn't remove the display of pop-up. So you need to remove it to solve this problem 
![11](https://user-images.githubusercontent.com/41991629/98460102-f21b5800-21a9-11eb-97f2-f035b315a447.png)

Reply:
I didn't notice.  Thanks👌👍 
On it right away.